core/storage/base_model/gae_models_test.py

```diff
@@ -332,9 +335,11 @@ def mock_put(self):
             if self.last_updated is None:
                 self.last_updated = datetime.datetime.utcnow()
 
-            # TODO(#13581): Fix the code so that the pylint: disable below can
-            # be removed.
-            super(base_models.BaseHumanMaintainedModel, self).put() # pylint: disable=bad-super-call
+            # We have not used BaseHumanMaintainedModel.put() because it is
+            # not implemented and throws error rather than saving the changes.
+            # We have used BaseModel.put() because we wanted the
+            # changes made above to be saved.
```

Comment:
`We are using BaseModel.put() to save the changes to the datastore since the put() method which TestBaseHumanMaintainedModel class inherits from BaseHumanMaintainedModel raises NotImplementedError.`